idea_town/frontend/static-src/app/views/details-view.js

```diff
@@ -0,0 +1,83 @@
+import Model from 'ampersand-model';
+import Collection from 'ampersand-collection';
+import CollectionRenderer from 'ampersand-collection-view';
+
+import BaseView from './base-view';
+import template from '../templates/details-view';
+import ContributorView from './contributor-view';
+
+const Contributors = Collection.extend({
+  model: Model.extend({
+    extraProperties: 'allow'
+  })
+});
+
+export default BaseView.extend({
+  _template: template,
+
+  bindings: {
+    'model.modified_date': {
+      type: function(el, value) {
```

Comment:
Getting an eslint warning here:
```
20:13  warning  Missing function expression name  func-names
```
Not sure I actually like this particular rule. Helps sometimes in debugging, but usually feels like boilerplate to me. But, we should either resolve this warning or disable the rule in `.eslintrc`